modules/ocf_mirrors/manifests/projects/devuan.pp

```diff
@@ -9,11 +9,7 @@
       recurse => true;
   }
 
-  ocf_mirrors::monitoring { 'devuan':
-    type          => 'unix_timestep',
-    upstream_host => 'https://www.devuan.org/#download',
-    ts_path       => 'TIME';
-  }
+# Looks like devuan doesn't have a timestamp file, so omitted ocf_mirrors::monitoring
```

Comment:
it doesn't need to block this PR, but we may want to contact them and ask for this to be added. we've done this with other distros in the past IIRC.